### DISCUSSION TITLE
Welcome to Community-Fibre-WHW03CFv2 Discussions!

### DISCUSSION THREAD

**Discussion by ishi0** [ishi0/Community-Fibre-WHW03CFv2#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by murobolo** [comment]
Hello
I uploaded Community-Fibre-WHW03CFv2 to the router, the installation went through, it restarted but the original firmware 1.0.11.208868 appeared again.
Where am I doing wrong??
Thanks for the info
M.

**Reply by ishi0** [reply]
Doesn't sound like you've installed the OpenWRT firmware and followed the steps through. Read through the Installation wiki page please and then report back.

Thanks

**Comment by murobolo** [comment]
I am sending a picture.  After this step, everything will return to its original state.  I tried it many times.  any idea Thank you.MuroOdoslané z iPhonuDňa 5. 2. 2023 o 0:02 užívateľ ishi0 ***@***.***> napísal:﻿
Doesn't sound like you've installed the OpenWRT firmware and followed the steps through. Read through the Installation wiki page please and then report back.
Thanks

**Comment by aadil-sheth** [comment]
I was following the steps and when installing the OpenWRT firmware, the router restarts and the Linksys firmware is still installed, I have tried installing the OpenWRT firmware multiple times, have waited longer than 5 minutes but it still doesn't install

Any advice?

**Comment by Zxnpool** [comment]
Can you use the latest openwrt or does it have to be the one on this github?

**Reply by jackoboy9** [reply]
I too would like to know this.

**Reply by arborluke** [reply]
Me too!

**Reply by Zxnpool** [reply]
Sorry for the delay in getting to you. Yeah it works! It works like normal! 

But I haven't tried to install linksys (non community fibre) firmware, just left it as OpenWRT

**Reply by Zxnpool** [reply]
I just got the specific one from the OpenWRT website I think (its been 2 months so I'm hazy) and I used that to install OpenWRT. My usecase required me to keep OpenWRT so I left it like that.

Idk if you can get the official updated linksys firmware though from the openWRT build from their website. I haven't tried.
